<!DOCTYPE html>
<html lang="en">

<head>

    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
    <meta name="description" content="">
    <meta name="author" content="">

    <title>jovo</title>

    <!-- Bootstrap Core CSS -->
    <link href="css/bootstrap.min.css" rel="stylesheet">
    <!-- font awesome -->
    <link rel="stylesheet" href="font-awesome/css/font-awesome.css">

    <!-- Custom CSS -->
    <link href="css/main.css" rel="stylesheet">

    <!-- ocp icon -->
    <link rel="shortcut icon" href="ocp_main3.ico">

    <!-- bower:css -->
    <!-- <link rel="stylesheet" href="/content/themes/jhu_id/bower_components/normalize-css/normalize.css"> -->
    <link rel="stylesheet" href="fonts/gentona/gentona.css">
    <link rel="stylesheet" href="fonts/titling-gothic/titling-gothic.css">
    <link rel="stylesheet" href="fonts/quadon/quadon.css">
    <link rel="stylesheet" href="fonts/arnhem/arnhem.css">
    <!-- endbower -->

    <!-- HTML5 Shim and Respond.js IE8 support of HTML5 elements and media queries -->
    <!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
    <!--[if lt IE 9]>
        <script src="https://oss.maxcdn.com/libs/html5shiv/3.7.0/html5shiv.js"></script>
        <script src="https://oss.maxcdn.com/libs/respond.js/1.4.2/respond.min.js"></script>
    <![endif]-->

</head>

<!-- The #page-top ID is part of the scrolling feature - the data-spy and data-target are part of the built-in Bootstrap scrollspy function -->

<body id="page-top" data-spy="scroll" data-target=".navbar-fixed-top">

    <!-- Navigation -->
    <nav class="navbar navbar-default navbar-fixed-top top-nav-collapse" role="navigation">
        <div class="container">
            <div class="navbar-header page-scroll">
                <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-ex1-collapse">
                    <span class="sr-only">Toggle navigation</span>
                    <span class="icon-bar"></span>
                    <span class="icon-bar"></span>
                    <span class="icon-bar"></span>
                </button>
                <a class="navbar-brand page-scroll" href="#page-top">jovo.me</a>
            </div>

            <!-- Collect the nav links, forms, and other content for toggling -->
            <div class="collapse navbar-collapse navbar-ex1-collapse">
                <ul class="nav navbar-nav">
                    <!-- Hidden li included to remove active class from philosophy link when scrolled up past philosophy section -->
                    <li class="hidden">
                        <a class="page-scroll" href="#page-top"></a>
                    </li>
                    <li>
                        <a class="page-scroll" href="#philosophy">philosophy</a>
                    </li>
                    <li>
                        <a class="page-scroll" href="#research">research</a>
                    </li>
                    <li>
                        <a class="page-scroll" href="#code">code</a>
                    </li>
                    <li>
                        <a class="page-scroll" href="#links">links</a>
                    </li>
                    <li>
                        <a class="page-scroll" href="#teaching">teaching</a>
                    </li>
                    <li>
                        <a class="page-scroll" href="#contact">contact</a>
                    </li>
                    <li>
                        <a class="page-scroll" href="#hiring">hiring</a>
                    </li>
                    <li>
                        <a class="fa fa-external-link" href="https://www.dropbox.com/s/xg96unmbvjurwa2/jovo_cv.pdf?dl=0" target="_blank">cv</a>
                    </li>
                    
                    
                </ul>
            </div>
            
        </div>
        <!-- /.container -->
    </nav>



    <!-- about me  -->
    <section id="about me" class="first-section">
        <div class="container">
            <div class="item row">
                <div class="col-md-2 col-sm-10 col-xs-12">
                    <h1> about me </h1>
                    <img class="img-responsive" src="jovo.png"/>
                </div>

                <div class="desc col-md-10 col-sm-10 col-xs-12">
                   <h1 class="lineup-hack" style="opacity: 0">me</h1>
                   <!-- ^ you better look at this comment jovo because this css hack makes it line up -->
                   <p> I am an Assistant Professor in the Department of <a href="http://www.bme.jhu.edu/">Biomedical Engineering</a> and the <a href="http://www.icm.jhu.edu/">Institute for Computational Medicine</a> at <a href="https://www.jhu.edu/">Johns Hopkins University</a>.  I sit at the <a href="http://cis.jhu.edu/">Center for Imaging Science</a> in Clark Hall at the Homewood campus.  I also have an appointment in the <a href="http://idies.jhu.edu/">Institute for Data Intensive Engineering and Sciences</a>.  My work focuses largely on big and wide data, especially neuroscience, focusing on statistics of brain graphs (connectomes).  I co-founded the <a href="http://ocp.me">Open Connectome Project</a> (OCP) with my brother R. Jacob Vogelstein and <a href="http://hssl.cs.jhu.edu/~randal/">Randal Burns</a>, Associated Professor in the Department of Computer Science at Johns Hopkins University. Recently it has spawned <a href="http://neurodata.io">NeuroData</a> which is the most of several complementary projects, including  OCP, Open Synaptome Project, and more.
                    We run a very vertical group, with people working at all levels of analysis, ranging from data collection to analysis and interpretation.  We are always looking for new collaborators and team members.  Please <a href="mailto:jovo@jhu.edu">inquire</a> if you are interested. 
                    </p>
                </div>
            </div>
        </div>
    </section>



    <!-- philosophy Section -->
    <section id="philosophy" class="generic-section">
        <div class="container">
            <div class="row">
                <div class="col-lg-12">
                    <h1>philosophy</h1>
                    <p> This world is already an incredibly beautiful and lovely place to live (at least for me, for now).  We are motivated by a desire to make it even better for all of us, as well as our descendants, and our fellow inhabitants.  We do so via searching for <em>patterns</em>. Specifically, we seek patterns in our physical worlds (e.g., our bodies, our brains) as well as our mental worlds (e.g., our perceptions, experiences, memories, thoughts, emotions, psychiatric conditions).  More importantly, we seek to understand patterns in our mental worlds <em>in terms of</em> our physical worlds. Our hope and belief is that via developing a deeper understanding of the links between these worlds, we will be able to bring them into greater alignment. A primary motivating factor is that all humans/animals have brains and therefore, such ideas could directly benefit all of animalkind. Thus, all of our research products are freely available to all.  For more  details see my 
                    <a href="https://www.dropbox.com/s/xg96unmbvjurwa2/jovo_cv.pdf?dl=0" target="_blank">cv</a>.
                </p>
                </div>
            </div>
        </div>
    </section>

    <!-- research Section -->
    <section id="research" class="generic-section">
        <div class="container">
            <div class="row">
                <div class="col-lg-12">
                    <h1>research</h1>
                    <p>The fundamental driving force of science is the discovery of latent structure that converts myriad disparate data points into understanding. In the 21st century, the growth of data acquisition has greatly outpaced the growth of data analysis, rendering current computational and statistical tools insufficient for extracting meaning from large datasets. Neuroscience is particularly susceptible to these big data challenges, as neuroexperimentalists have devised methods of collecting terabytes of data per hour. Without statistical and computational frameworks for analysis and organization of these data, the field will be unable to fully reap the benefits of this enormous potential. Our expertise in (i) statistical science (ii) computer science, and (iii) neuroscience  enables us to fill this gap. Below we elaborate on these three threads.
                    </p>

                    <h2>statistical science</h2>

                    <p>Statistical sciences have developed a beautiful corpus of knowledge on inference in various settings over the last 100 years.  However, in recent years, new experimental and data collection methods have yielded datasets that no longer fit within the assumptions upon which that body of literature rests.  We therefore  focus  on building statistical methods for 21st century data, including
                    <ul>
                        <li><strong>time-series</strong> sequential monte carlo, non-negative deconovlution, variational inference, and high-dimensional system identification;</li>
                        <li><strong>graphs</strong> inference of graphs, supervised and unsupervised vertex clustering, similarity and dissimilarity metrics, and population theory including graph classification and joint embedding;</li>
                        <li><strong>shapes</strong> extending large deformation diffeomorphic metric mapping (<a href="http://cis.jhu.edu/software/">LDDMM</a>) to work on multimodal and big data;</li>
                        <li><strong>vectors</strong> high-dimensional linear and non-linear classification, optimal design, signal detection, density estimation, two-sample testing, and dependence discovery.</li>
                    </ul>
                    </p>

                    <h3>computational science</h3>

                    <p>Statistical methods require computational implementations. Moreover, the raw data of the 21st century is not typically in a form immediately amenable to statistical analysis; rather, various data wrangling/munging/pre-processing is typically beneficial.   We therefore develop computational tools to enable principled statistical inference on such data.  This includes:

                    <ul>
                        <li><strong>databases</strong> for managing, (i)
                        <em>spatial databases</em> for petascale 3D, 4D (multispectral or time-series), and 5D data (both);, (ii) <em>metadata databases</em> annotations and metadata, particularly for neuroanatomy; and (iii) <em>grute databases</em> for populations of massive graphs with rich attributes;</li> 
                        <li><strong>distributed computer vision pipelines</strong> for electron microscopy,  functional magnetic resonance imaging, diffusion tensor imaging, and object detection in both MATLAB and Python;</li>
                        <li><strong>parallel R</strong> for billion vertex graphs including various graph traversal, vertex clustering, and matrix applications such as svd, pca, k-means, and linear classification;  </li>
                        <li><strong>interactive visualizations</strong> including for multispectral annotated 3D+ big images, graphs, and vectors.</li>
                    </ul>
                    </p>

                    <h3>neuroscience</h3>

                    <p>The above described tools, both statistical and computational, are designed and built in the service of answering fundamental questions in neuroscience, both basic and clinical. In basic research, I have begun deciphering the connectome in various ways [23, 40, 41]. I recently utilized the statistical tools I developed to learn a mapping from Drosophila larva neuron types to learned behaviors [42] (see Figure), and combined our statistical and computational tools to test (and refute) Peters rule governing connections amongst neighboring neurons [35]. Translationally, I have developed tools to detect Alzheimers disease [43] and multiple sclerosis lesions [44]. More generally, I quantified the predictive capacity of personal genome sequencing to discover that for any given disease, personal genomics would often not be very useful, although across genes it could yield utility [45, 46]. This discovery led to a debate on the validity of whole genome sequencing (see here, here, here, and here).</p>


                    <ul>
                      <li><strong>human connectomics: </strong> Even the simplest questions about human connectomes (brain-graphs) - such what is the mean connectome, and are these two populations of connectomes different - remain unanswered.  In part, this is due to a lack of theoretically justified tools with resulting scalable code and appropriate data to answer them.  We have developed fundamental theory (e.g., <a href="http://arxiv.org/abs/1403.7249" target="_blank">Tang14a</a>), and scalable code (<a href="http://www.flashgraph.net/" target="_blank">FlashGraph</a>), that will now enable us to rigorously answer these questions.  We are looking for students who would like to extend and apply these methods to extant collected and pre-processed human connectomes (available <a href="http://openconnecto.me/data/public/graphs/" target="_blank">here</a>), to get some quantitative answers. A video describing some of the related data can be found <a href="https://www.youtube.com/watch?v=TxlV50P6NEI" target="_blank">here</a>.</li>

                       <li><strong>spike detection</strong> from multi-channel and online e-phys data, as well as calcium imaging data;</li>
 
                      <li><strong>synaptic diversity: </strong> All brains are composed of neurons, and the connections between them, called synapses.  All brain function is therefore at least partially determined by these synapses, and yet, we are woefully clueless with regard to their diversity and functional properties. We do know that each synapse is a complex composition of hundreds of proteins, all acting in concert together (read <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3670986/" target="_blank">O'Rourke12a</a> for a nice review).  We have the data, as well as the methodological tools to enable exploring this diversity.  We are looking for students who would like to extend and apply these methods to the extant collected and pre-processed data sets (available <a href="http://www.openconnectomeproject.org/#!array-tomography/c1z9t" target="_blank">here</a>), to reveal previously unknown but fundamental principles of neural computation. Videos describing these datasets can be found <a href="http://smithlab.stanford.edu/Smithlab/AT_Movies.html" target="_blank">here</a>.</li>

                      <li><strong>electron microscopy connectomics:</strong> Microcircuits of the brain enable computational capabilities far exceeding our greatest modern supercomputers, yet they fit in little skulls weighing less than a grapefruit.  It is widely believe that cracking the neural circuitry code will reveal mysteries of the brain.  To do so, however, we need petascale computer vision methods, such as three-dimensional scene parsing, as well as statistical tools to analyze these large and complex datasets.  We are looking for students who would like to extend and apply these methods to the extant collected and pre-processed data sets (available <a href="http://www.openconnectomeproject.org/" target="_blank">here</a>).  Videos describing these datasets can be found <a href="http://www.openconnectomeproject.org/#!s1-videos/cn89" target="_blank">here</a> and <a href="http://www.openconnectomeproject.org/#!mouse-v1-video/cloz" target="_blank">here</a>.</li>

                      <li><strong>connectome coding: </strong> Understanding the relationship between brain properties (for example, connectivity) and mental properties (for example, memories) remains one of the greatest challenges in the 21st century.  It is also an example of a class of cutting-edge machine learning problems, sometimes called multi-modal learning.  We have already developed some tools (e.g., <a href="http://www.cis.jhu.edu/~parky/CEP-Publications/BJPS188.pdf" target="_blank">JOFC</a>) and have pre-processed some datasets (e.g., <a href="http://fcon_1000.projects.nitrc.org/indi/retro/NEO2012.html" target="_blank">Adelstein et al, 2001</a>), we are looking for students to extend and apply these methods to uncover, for example, relationships between brain networks and personality types.</li>
                      <li><strong>Your project here!</strong></li>
                    </ul>

                    <p>For more information about any of these projects, or to tell me about your awesome new project, <a href="mailto:jovo@jhu.edu">email me</a>, or swing by <a class="page-scroll" href="#contact">my office</a> sometime!</p>
                </div>
            </div>
        </div>
    </section>

    <section id="code" class="generic-section">
        <div class="container">
            <div class="row">
                <div class="col-lg-12">
                    <h1>code</h1>
                    <p>Essentially every project I embark on includes writing some code, if only to buttress the theoretical results with numerical examples.  Moreover, as an adherent to the philosophy of open science, code I write is always open source by the time of submission, and often sooner.  Thus, searching my <a href="https://www.dropbox.com/s/xg96unmbvjurwa2/jovo_cv.pdf?dl=0" target="_blank">cv</a> is the most reliable method for discovering all of the code bases associated with any project I am a part of.  You can also check various github accounts that I regularly contribute to, including <a href="https://github.com/jovo" target="_blank">my own</a> and <a href="https://github.com/openconnectome/" target="_blank">open connectome project</a>'s, which we started. We also develop <a href="http://www.flashgraph.net/" target="_blank">FlashGraph</a> these days. Nevertheless, here I will try to post the codes from various projects that I think are potentially useful for other applications.  </p>

                    <h3 id="research-topics"><strong>time series analysis</strong></h3> 
                    <ol>
                        <li>Fast Nonnegative Deconvolution for Spike Train Inference from Population Calcium Imaging. <a href="https://github.com/jovo/fast-oopsi/raw/master/fast-oopsi.pdf" target="_blank">manuscript</a>, <a href="https://github.com/jovo/fast-oopsi/raw/master/code/functions/fast_oopsi.m" target="_blank">code</a>, and <a href="https://github.com/jovo/fast-oopsi" target="_blank">repo</a> .</li>
                        <li>Spike inference from calcium imaging using sequential Monte Carlo methods.  <a href="https://github.com/jovo/smc-oopsi/raw/master/manuscript/VogelsteinPaninski09.pdf" target="_blank">article</a>, <a href="https://github.com/jovo/oopsi" target="_blank">code</a>.</li>
                        <li> Real-Time Inference for a Gamma Process Model of Neural Spiking. <a href="http://papers.nips.cc/paper/5061-real-time-inference-for-a-gamma-process-model-of-neural-spiking" target="_blank">manuscript</a>, <a href="https://github.com/decarlson/opass" target="_blank">code</a>.</li>
                    </ol>
                    <h3 id="research-topics"><strong>graph analysis</strong></h3> 
                    <ol>
                        <li>Graph Classification using Signal Subgraphs: Applications in Statistical Connectomics. <a href="http://arxiv.org/abs/1108.1427" target="_blank">arxiv</a>, <a href="https://github.com/jovo/MatGraphStat/tree/master/SignalSubgraphClassifier" target="_blank">code</a>, <a href="https://github.com/jovo/signal-subgraph-classifier" target="_blank">repo</a>, <a href="http://www.computer.org/csdl/trans/tp/2013/07/ttp2013071539-abs.html" target="_blank">article</a>.</li>
                        <li>Fast Approximate Quadratic Programming for Large (Brain) Graph Matching. <a href="http://arxiv.org/abs/1112.5507" target="_blank">arxiv</a>, <a href="https://github.com/jovo/MatGraphStat" target="_blank">code</a>.</li>
                        <li>Seeded Graph Matching Via Joint Optimization of Fidelity and Commensurability. <a href="http://arxiv.org/abs/1401.3813" target="_blank">arxiv</a>, <a href="https://github.com/igraph/xdata-igraph/blob/develop/interfaces/R/igraph/R/sgm.R" target="_blank">code</a>.</li>
                        <li>Robust Multimodal Graph Matching: Sparse Coding Meets Graph Matching. <a href="http://papers.nips.cc/paper/4925-robust-multimodal-graph-matching-sparse-coding-meets-graph-matching">nips</a>, <a href="http://arxiv.org/abs/1311.6425" target="_blank">arxiv</a>, <a href="http://www.fing.edu.uy/~mfiori/code/code_graph_matching.zip">code</a>. </li>
                        <li>  DELTACON: Measuring Connectivity Differences in Large Networks. <a href="http://dx.doi.org/10.1137/1.9781611972832.18">article</a>. <a href="http://arxiv.org/abs/1304.4657" target="_blank">arxiv</a>, <a href="http://www.cs.cmu.edu/~dkoutra/CODE/deltacon.zip" target="_blank">code</a>.
                        <li>MR Connectome Automated Pipeline. <a href="http://arxiv.org/abs/1111.2660">arxiv</a>, <a href="http://nitrc.org/projects/mrcap" target="_blank">code</a>, <a href="http://ieeexplore.ieee.org/xpl/articleDetails.jsp?tp=&arnumber=6173097&contentType=Journals+%26+Magazines&sortType%3Dasc_p_Sequence%26filter%3DAND%28p_IS_Number%3A6173090%29" target="_blank">abstract</a>, <a href="http://db.tt/BMEVlrpE" target="_blank">pdf</a>.</li>
                    </ol>
                </div>
            </div>
        </div>
    </section>

    <section id="links" class="generic-section">
        <div class="container">
            <div class="row">
                <div class="col-lg-12">
                    <h1>links</h1>
                    <a href="http://scholar.google.com/citations?hl=en&user=DWPfdT4AAAAJ&sortby=pubdate&view_op=list_works&gmla=AJsN-F7eFaJivagMCwgIbNf4cOwHKPj8aFcIANb8rku8bPmHR88ZxVs8dbsn68pjjibAZ9I4KrtOJDCVC411dnpjg51-NPcMQ6SrgdFh3NXdXyjq9dimPVg" target="_blank">jovo@google scholar</a><br>
                    <a href="http://www.ncbi.nlm.nih.gov/myncbi/browse/collection/40260830/?sort=date&direction=ascending" target="_blank">jovo@pubMed</a><br>
                    <a href="http://arxiv.org/find/all/1/au:+Vogelstein_J/0/1/0/all/0/1" target="_blank">jovo@arXiv</a><br>
                    <a href="https://github.com/jovo" target="_blank">jovo@github</a><br>
                    <a href="http://www.openconnectomeproject.org/" target="_blank">open connectome project</a><br>
                    <a href="https://www.dropbox.com/s/xg96unmbvjurwa2/jovo_cv.pdf?dl=0" target="_blank">cv</a>
                </div>
            </div>
        </div>
    </section>

    <section id="teaching" class="generic-section">
        <div class="container">
            <div class="row">
                <div class="col-lg-12">
                    <h1>teaching</h1>
                    <ul>
                    
                    <li><a href="https://github.com/Upward-Spiral-Science/" target="_blank">The Art of Data Science (Spring 2016)</a></li>
                    
                    <li>Introduction to Computational Medicine I (Fall 2015)</li>
                    
                    <li><a href="https://github.com/Statistical-Connectomics-Sp15/intro" target="_blank">Statistical Connectomics (Spring 2015)</a></li>
                    </ul>
                </div>
            </div>
        </div>
    </section>

    <!-- Contact Section -->
    <section id="contact" class="generic-section">
        <div class="container">
            <div class="row">
                <div class="col-lg-12">
                    <h1>contact</h1>
                    <p>jovo [at symbol] jhu [dot] edu<br>
                        <br>
                            Joshua T. Vogelstein<br>
                            Center for Imaging Science<br>
                            Clark Hall, Rm 317C<br>
                            3400 N. Charles St<br>
                            Baltimore, MD 21218
                    </p>
                </div>
            </div>
        </div>
    </section>

    <!-- Hiring Section -->
    <section id="hiring" class="generic-section">
        <div class="container">
            <div class="row">
                <div class="col-lg-12">
                    <h1>hiring</h1>
<!--                     <h2>for all applicants</h2>
 -->                   <p>
                    NeuroData is always hiring exceptional individuals at all levels. There are only a few requirements: (1) you are excited to work closely with a team of diverse thinkers, including computer scientists, biomedical engineers, and neuroscientists, (2) you are will to commit at least 2-3 years to the project, and (3) you understand that we do open science - this means that all code that we write is open sourced and therefore run by other people (one implication of this is that our code is always tested and documented). Assuming you fit those conditions, please read the below specialized instructions.
                    </p>

                    <p>our work spans across three disparate fields: applied mathematics & statistics, computer science, and neuroscience.  we are always looking for exceptional people that live in any of those fields, and/or across them.  our statistical work includes developing new methods, proving theorems about them, and demonstrating their real world utility.  our computational work involves making algorithms scale to work with petascale data, including petascale images, graphs, and arrays.  our neuroscience work involves applying our novel and scalable methods to answer the most pressing problems that are not adequately answerable without such tools.</p>

                    <p>we are currently particularly looking for people to work on one of the following three projects
                    <ul>
                        <li><strong>graph statistics:</strong>  we will be (i) writing reference implementations in R for a number of graph statistical methods for which we do not currently have said implementations, (ii) developing new graph statistical theory and methods, and (iii) applying said methods to neurographs. All reference implementations will be incorporated into <a href="http://flashx.io">FlashX</a>. We will write papers on graph statistics aimed at statistics and machine learning audiences, as well as some aimed at neuroscientists.</li>
<!--                         <li><strong>shape statistics:</strong> like graph statistics, except for large shapes of various genuses.</li>
                        <li><strong>vector statistics:</strong> like graph & shape statistics, except for large vectors.</li>
 -->                        <li><strong>computational anatomy and physiology:</strong> we will be (i) writing code using the NeuroData infrastructure to extract neuroanatomical objects of interest (e.g., cells, synapses, regions), and (ii) developing and deploying methods for scalable statistical analyses for these objects, such as 3D point processes for 100 million points. Some of this work is explained in more detail <a href="https://github.com/openconnectome/neurodata.io/files/7260/postdoc.pdf">here.</a> </li>
                        <li><strong>human MRI:</strong>  will also be writing code using the NeuroData infrastructure, processing lots of open source brains, and making discoveries and writing papers using graph statistics, spatial statistics, and more. One example of this work is the MRI-to-Graphs pipeline, lovingly refered to as <a href="http://m2g.io">m2g.</a> </li>

                    </ul>

                    </p>

                    <p>regardless of which position you would like to fill, please fill out <a href="http://goo.gl/forms/tEXaL2N2yu">this form</a>, and when complete, please email your cv and transcript to jovo at jhu dot edu.  see below for job qualifications:</p>

                    <ul>
                        <li><strong>research associate:</strong> an undergraduate degree in a related field, eg, biomedical engineering, neuroscience, or computer science.</li>
                        <li><strong>current JHU undergrads:</strong> you'll need to commit to a minimum of 10 hrs per week for at least 2 years, plus a GPA of >3.5 including engineering classes.</li>
                        <li><strong>current JHU masters students:</strong>  if you are planning on doing a PhD after you MS, fill out the form and email me :)</li>
                        <li><strong>current JHU phd students:</strong>  if you'd like to do your dissertation in our group, fill out the form and email me :) </li>
                        <li><strong>potential JHU graduate students: </strong> we are always accepting new students. the best plan is to apply to the <a href="http://www.bme.jhu.edu/graduate/phd/prospective-students">BME PhD program</a>, applications are typically due in late november/early december. i can also advise students in other departments, including biostats, neuroscience, and computer science.</li>
                        <li><strong>postdocs:</strong> we are especially always hiring postdocs! please fill out the form and email me!</li>
                    </ul>

                </div>
            </div>
        </div>
    </section>



    <section class="copyright">
        <div class="container">
            <div class="row">
                <div class="col-lg-12">
                    <p class="copyright">designed by <a href="http://ronboger.me" target="_blank">ronnyb</a>, jovo, <a href="http://ikiar.ca/">greg kiar</a>, and <a href="http://jordan.matelsky.com/">jordan matelsky</a>. the website of dr. joshua vogelstein &copy; jovo.me 2015</p>                    
                </div>
            </div>
        </div>
    </section>

    <!-- jQuery -->
    <script src="js/jquery.js"></script>

    <!-- Bootstrap Core JavaScript -->
    <script src="js/bootstrap.min.js"></script>

    <!-- Scrolling Nav JavaScript -->
    <script src="js/jquery.easing.min.js"></script>
    <script src="js/main.js"></script>

</body>

</html>
